apps/coscrad-frontend/src/components/resources/vocabulary-lists/vocabulary-list-detail.full-view.presenter.tsx

```diff
@@ -89,24 +90,25 @@ export const VocabularyListDetailFullViewPresenter = ({
     const selectedEntries = filterEntriesForSelectedTerms(entries, filterWithoutNullAndUndefined);
 
     return (
-        <div data-testid={id}>
-            <h1>{formatBilingualText(name, nameEnglish)}</h1>
-            <div>
+        <Typography sx={{ textAlign: 'center' }} component={'div'} data-testid={id}>
```

Comment:
Just a detail, but I think this can also be `component="div"`.  But it looks like you're using `Typography` as a container?  Rather than to specify a font style.  I would use `Box` here instead.  Same with the instances of `Typography` below.  Anywhere we're doing layout we should use things like `Container`, `Box`, `Grid`, or if you're doing a single column of elements, `Stack`.

Reply:
Resolved.